src/dsnotificationinterface.cpp

```diff
@@ -74,14 +74,15 @@ void CDSNotificationInterface::UpdatedBlockTip(const CBlockIndex *pindexNew, con
 
 void CDSNotificationInterface::SyncTransaction(const CTransactionRef& tx, const CBlockIndex* pindex, int posInBlock)
 {
-    llmq::quorumInstantSendManager->SyncTransaction(tx, pindex, posInBlock);
-    llmq::chainLocksHandler->SyncTransaction(tx, pindex, posInBlock);
+    // TODO when the old InstantSend system is removed, also remove this whole method and all the surrounding compatiblity code
```

Comment:
I think we can actually drop `SyncTransaction` here already since it's not used for anything else but for legacy InstantSend, see 4bef5dd2ecabb5fb1729e5146701dddf255939ef.

Reply:
Thanks, applied the commit and pushed :+1: